src/app/users/users-profile/users-profile-dialog.component.ts

```diff
@@ -18,10 +20,15 @@ export class UserProfileDialogComponent {
 
   name: string;
   planetCode: string;
+  isHidden: boolean;
 
-  constructor(@Inject(MAT_DIALOG_DATA) public data: any) {
+  constructor(
+  @Inject(MAT_DIALOG_DATA) public data: any,
+  private userService: UserService
+  ) {
     this.name = data.member.name;
     this.planetCode = data.member.userPlanetCode;
+    const currentUser = this.userService.get();
+    this.isHidden = !currentUser.isUserAdmin;
```

Comment:
Please use [ViewChild](https://angular.io/api/core/ViewChild) to get the `editable` property from `UsersProfileComponent`.  That is based on some more conditions in addition to `isUserAdmin` which should also be used to show/hide the link to the user profile page.

Reply:
I cannot seem to get it working using editable property working, the state of editable seems to be always `false` even in the profileView function in the users-profile.component.ts.